src/dbnode/mkdocs.yml

```diff
@@ -1,47 +0,0 @@
-site_name: M3DB Documentation
```

Comment:
was this just stale?

Reply:
Yeah, it was just floating there, not used anywhere.